vaults/test.yml

```diff
@@ -1,12 +1,43 @@
 ---
-cloud_token: 'this is the one from console.redhat.com'
-offline_token: 'this is the one linked below about api token'
-rh_username: 'redhat user login (this is used to attach your subs to controller)'
-rh_password: 'password for redhat account'
-root_machine_pass: 'password for root user on builder (if not root user more changes will need to be made)'
-ah_token_password: 'this will create and use this password can be generated'
-controller_api_user_pass: 'this will create and use this password can be generated'
-controller_pass: 'admin account pass for controller, if none is given it will default to Password1234!'
-ah_pass: 'hub admin account pass, if none is given it will default to Password1234!'
-vault_pass: 'the password to decrypt this vault'
+# This token is used to synchronise certifified and validated collections from
+# console.redhat.com.
+# To generate an AH token, go to the url:
+# https://console.redhat.com/ansible/automation-hub/token and click on
+# "Load Token" in the "Offline token" chapter.
+cloud_token: ''
+
+# This token use Red Hat's API to download AAP
+# Generate this token here: https://access.redhat.com/management/api
+# Keep in mind that the token will expire if it is not used for 30 days.
+offline_token: ''
+
+# Red Hat account login (this is used to attach your subs to controller)
+rh_username: ''
+
+# Red Hat account password
+rh_password: ''
+
+# The password for the root user on builder (using a non-root user would require
+# further changes)
+root_machine_pass: ''  # We disallow login via password on sshd, so we will
```

Comment:
I think you want to remove this second comment about disallowing due to that being your env

Reply:
You're right. I forgot to remote those comments about my SSHD settings. It's done.